erpnext/projects/doctype/task/task.json

```diff
@@ -367,13 +370,36 @@
    "fieldname": "billable",
    "fieldtype": "Check",
    "label": "Billable"
+  },
+  {
+   "fetch_from": "project.project_team",
+   "fieldname": "team",
+   "fieldtype": "Link",
+   "label": "Team",
+   "options": "Project Team",
+   "read_only": 1
+  },
+  {
+   "depends_on": "eval:doc.team",
+   "fieldname": "assign_to",
+   "fieldtype": "Link",
+   "in_list_view": 1,
+   "label": "Assign To",
+   "options": "Employee"
+  },
```

Comment:
- Change this to Table MultiSelect, since multiple users can be assigned to a project
- If this field is set, you will have to add assignment to the user, ie create a `ToDo` for that user.